templates/addon/Jenkinsfile.j2

```diff
@@ -1 +1 @@
-buildPlugin(version: "Leia", platforms: {{ ['android-armv7', 'android-aarch64', 'ios-armv7', 'ios-aarch64', 'osx-x86_64', 'ubuntu-ppa', 'windows-i686', 'windows-x86_64']|reject("in", libretro_repo.exclude_platforms)|list()|tojson() }})
+buildPlugin(version: "Matrix", platforms: {{ ['android-armv7', 'android-aarch64', 'ios-armv7', 'ios-aarch64', 'osx-x86_64', 'ubuntu-ppa', 'windows-i686', 'windows-x86_64']|reject("in", libretro_repo.exclude_platforms)|list()|tojson() }})
```

Comment:
`ios-armv7` doesn't exist for Kodi master anymore. Also, `tvos-aarch64` should be added

Reply:
Do we even need to specify the platforms at all?

Reply:
I modelled this on https://github.com/kodi-game/game.libretro/commit/ea6c8e20e554b4c6f0e9cb5304db4568d8d08ffc, and it looks like I didn't fully update this file. Should be simpler now without the platforms.